bcda/cclf/cclf.go

```diff
@@ -81,9 +81,9 @@ func importCCLF0(fileMetadata *cclfFileMetadata) (map[string]cclfFileValidator,
 	defer r.Close()
 
 	const (
-		fileNumStart, fileNumEnd           = 0, 13
-		totalRecordStart, totalRecordEnd   = 35, 55
-		recordLengthStart, recordLengthEnd = 56, 69
+		fileNumStart, fileNumEnd           = 0, 7
+		totalRecordStart, totalRecordEnd   = 52, 63
+		recordLengthStart, recordLengthEnd = 64, 70
```

Comment:
With the start and end being 65 and 69 in the IP, should `recordLengthEnd` be 69?

Reply:
looks like I copied from Slack and got some whitespace that was extra.  I have changed it to 69.